firedrake/slate/slate.py

```diff
@@ -549,38 +590,40 @@ def __init__(self, tensor, coefficient):
             "coefficient function space."
         )
         super(Action, self).__init__()
-        self.tensor = tensor
-        self._acting_coefficient = coefficient
+        self.operands = tensor, coefficient
```

Comment:
Should I treat `operands` then to strictly mean the participating Slate objects? Technically the `coefficient` is an operand. I can just add back in the attribute `acting_coefficient` if that's nicer.